crates/bevy_ui/src/flex/mod.rs

```diff
@@ -14,32 +14,32 @@ use bevy_transform::components::Transform;
 use bevy_utils::HashMap;
 use bevy_window::{Window, WindowId, WindowScaleFactorChanged, Windows};
 use std::fmt;
-use stretch::{number::Number, Stretch};
+use taffy::{number::Number, Taffy};
 
 pub struct FlexSurface {
-    entity_to_stretch: HashMap<Entity, stretch::node::Node>,
-    window_nodes: HashMap<WindowId, stretch::node::Node>,
-    stretch: Stretch,
+    entity_to_sprawl: HashMap<Entity, taffy::node::Node>,
+    window_nodes: HashMap<WindowId, taffy::node::Node>,
+    sprawl: Taffy,
 }
 
-// SAFE: as long as MeasureFunc is Send + Sync. https://github.com/vislyhq/stretch/issues/69
+// SAFE: as long as MeasureFunc is Send + Sync. https://github.com/vislyhq/sprawl/issues/69
 // TODO: remove allow on lint - https://github.com/bevyengine/bevy/issues/3666
 #[allow(clippy::non_send_fields_in_send_ty)]
 unsafe impl Send for FlexSurface {}
 unsafe impl Sync for FlexSurface {}
 
 fn _assert_send_sync_flex_surface_impl_safe() {
     fn _assert_send_sync<T: Send + Sync>() {}
-    _assert_send_sync::<HashMap<Entity, stretch::node::Node>>();
-    _assert_send_sync::<HashMap<WindowId, stretch::node::Node>>();
-    // FIXME https://github.com/vislyhq/stretch/issues/69
-    // _assert_send_sync::<Stretch>();
+    _assert_send_sync::<HashMap<Entity, taffy::node::Node>>();
+    _assert_send_sync::<HashMap<WindowId, taffy::node::Node>>();
+    // FIXME https://github.com/vislyhq/sprawl/issues/69
```

Comment:
```suggestion
    // FIXME https://github.com/vislyhq/stretch/issues/69
```
same her with the link